refactor.rb

```diff
@@ -0,0 +1,15 @@
+class People
+	attr_reader :age, :phase
+	attr_accessor :name
+
+	def initialize(options={})
+		@phase = 3 # not sure 
+		@age = options.fetch(:age, 0)
+    @name = options.fetch(:name, "")
+	end
+
+	def offer_high_five   #not sure if this is appropiate cause it's a behavior
+    "High five!"
+  end
+
```

Comment:
My tab was giving me 4 spaces when I start and then I change the setting to 2 spaces but I forgot to reformat my code😭